CHANGELOG.md

```diff
@@ -13,6 +13,7 @@
 Replace `getUserName`, `getProfilePhoto` with new prototypes.
 
 - **Change:** Support Android 7 - API 24 as minimum version.
+- **Change:** Convert `UserInfoBridgeDispatcher` into interface.
```

Comment:
Could you also add a note about how they should update their code? i.e. something like: 
```suggestion
- **Change:** Convert `UserInfoBridgeDispatcher` into interface. Usages in your code of `object : UserInfoBridgeDispatcher()` should be changed to `object : UserInfoBridgeDispatcher`.
```

Reply:
updated.